src/frontend/query_protocol.ml

```diff
@@ -130,6 +130,7 @@ type _ t =
        | `Not_found of string * string option
        | `No_documentation
        ] t
+  | Type_definition : Msource.position -> (string option * Lexing.position) list t
```

Comment:
Maybe it should be called `Locate_type` (as we have `Locate` which does the same for values).

Reply:
Renamed